packages/web-react/src/components/Accordion/__tests__/AccordionHeader.test.tsx

```diff
@@ -7,6 +7,14 @@ import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
 import AccordionHeader from '../AccordionHeader';
 import AccordionItem from '../AccordionItem';
 
+jest.mock('../../../hooks', () => {
```

Comment:
**thought (if-minor):** There is a question about how to keep this DRY and import it or call from one place.
It is prohibited by the ESLint rule - https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-mocks-import.md